Makefile

```diff
@@ -207,6 +220,56 @@ $(BUILD_DIRECTORY)/electron-$(ELECTRON_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH
 		-s "$(TARGET_PLATFORM)" \
 		-o $@
 
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH)-app: \
+	package.json lib \
+	$(BUILD_DIRECTORY)/node-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/node_modules \
+	| $(BUILD_DIRECTORY)
+	mkdir $@
+	$(foreach prerequisite,$^,$(call execute-command,cp -rf $(prerequisite) $@))
+
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH).js: \
+	$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH)-app \
+	| $(BUILD_DIRECTORY)
+	./scripts/build/concatenate-javascript.sh -e $</lib/cli/etcher.js -o $@
+
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH): \
+	$(BUILD_DIRECTORY)/node-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/node_modules \
+	$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH).js \
+	| $(BUILD_DIRECTORY) $(BUILD_TEMPORARY_DIRECTORY)
+	./scripts/build/node-package-cli.sh -o $@ -l $< \
+		-n $(APPLICATION_NAME) \
+		-e $(word 2,$^) \
+		-r $(TARGET_ARCH) \
+		-s $(TARGET_PLATFORM)
+ifeq ($(TARGET_PLATFORM),win32)
```

Comment:
Adding an empty line before the platform check here would be good to improve readability & quick comprehension I think